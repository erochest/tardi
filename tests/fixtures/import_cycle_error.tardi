use: ../modules/import-cycle-1
