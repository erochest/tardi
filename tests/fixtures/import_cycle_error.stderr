Error: CompilerError(ImportCycleError("import-cycle-1"))
